client/src/components/layouts/DonorPage.tsx

```diff
@@ -5,9 +5,17 @@ import Navbar from '../organisms/Navbar'
 
 const DonorPage: FunctionComponent<Record<string, unknown>> = (props: React.PropsWithChildren<Record<string, unknown>>) => {
   return <div className="donor-page">
-    <div className="donor-page-header"><Navbar /></div>
-    <div className="donor-page-content" >{props.children}</div>
-    <div className="donor-page-footer"><Footer /></div>
+    <div className="donor-page-header">
+      <Navbar links={ [ { name: "TPC Main Website", link: "/donation-guidelines"  }, 
```

Comment:
Can we link "TPC Main Website" to `https://pregnancycentre.ca/`?

Reply:
done